src/Atmos/Model/turbulence.jl

```diff
@@ -241,14 +250,15 @@ end
 function diffusive!(::Vreman, orientation::Orientation,
                     diffusive::Vars, ∇transform::Grad, state::Vars, aux::Vars, t::Real)
   diffusive.turbulence.∇u = ∇transform.u
+  diffusive.turbulence.S = symmetrize(diffusive.turbulence.∇u)
```

Comment:
This is storing the same thing twice, which will increase our GPU memory usage. Possibly we should just be storing the symmeterized version?

Reply:
The Vreman subgridscale model seems to need both, however it computes the symmeterized version locally. However I currently need S to be accessible to do the Courant calculation. This would be simple if all the subgridscale models either typically stored only of the two tensors but this seems to vary depending on the choice of turbulence closure. It might be best in this case to possibly only store the non symmeterized version for all closures and have them compute the symmetrized version locally so that it is never saved before this is we must be certain it is not needed elsewhere.

Reply:
Recomputing simple algebraic quantities is almost alway preferential in general (loading data is the most expensive thing we do, so loading the same data multiple times in the same functions doesn't make sense).

When we start needing to do iterations to computing values or using special functions the picture changes a bit.

Reply:
It looks S isn't used outside of turbulence.jl I'm going to modify it so that only the non-symmeterized version is saved and S can be computed for each closure as needed.